### DISCUSSION TITLE
Moving an existing docker container behind reverse proxy

### DISCUSSION THREAD

**Discussion by vikasrawal** [nextcloud/all-in-one#3022] [Questions] [answered]
I am trying to move an existing nextcloud aio container behind a reverse proxy.  I am new to docker, and have learned about it over the last few weeks in the process of setting up nextcloud.

I am following the instructions given at: [https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md)

However, since I already have a container, "docker run" does not work. How can I change the settings of the existing container? Or is it recommended that I create a fresh container? Would that result in data loss?

V.

**Comment by vikasrawal** [comment]
I read somewhere that I could indeed delete the container and recreate it. I have gone ahead and done that in the hope that I would be able to restore the data. But I am now getting this error in the logs:

> Could not resolve the host nextcloud.com.
> Most likely the DNS resolving does not work.
> You should be able to fix this by following https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html
> Apart from that, there has been this: https://github.com/nextcloud/all-in-one/discussions/2065

I can't understand instructions in the dockerlabs.... link. I ahev tried instructions in the other link but that does not help.

Wouldd appreciate any help in fixing the reverse proxy issue and getting this back on its feet again.

Thanks.

**Comment by szaimen** [comment, marked as answer]
Does it work now and what exactly dont you understand in the provided link?
